livre/eq/Window.cpp

```diff
@@ -69,10 +69,12 @@ struct Window::Impl
                         pipe->getFrameData()->getVRParameters().getMaxGPUCacheMemoryMB();
         _textureCache.reset( new TextureCache( node->getTextureDataCache(),
                                                maxGpuMemory * LB_1MB, GL_LUMINANCE8 ));
-
-        const size_t computeThreads = 4;
+        const size_t renderThreads = 2;
+        const size_t computeThreads = 2;
         const size_t uploadThreads = 4;
```

Comment:
It was arguments in the first pipeline implementation, then I said lets do when we need it but would be nice to have, to set uploader threads or compte threads and see the advantages.